GeoHealthCheck/app.py

```diff
@@ -481,9 +483,10 @@ def add():
         APP.config, resource_type, url)
 
     if not success:
+        LOGGER.exception(message)
         flash(message, 'danger')
-        return redirect(url_for('add', lang=g.current_lang,
-                                resource_type=resource_type))
+        # return redirect(url_for('add', lang=g.current_lang,
+        #                         resource_type=resource_type))
```

Comment:
Yes, this was the error when initial "sniffing" failed, in the example WMS 1.3.0.